### DISCUSSION TITLE
info Using wasm build of next-swc

### DISCUSSION THREAD

**Discussion by Ebibeck** [vercel/next.js#49700] [Help] [answered]
### Summary

- info Using wasm build of next-swc
- warn Attempted to load @next/swc-win32-x64-gnu, but it was not installed
- warn Attempted to load @next/swc-win32-x64-msvc, but an error occurred: The specified module could not be found.

this is way out of my leauge....help please

**Comment by DanteTerry** [comment, marked as answer]
Just uninstall your current version of node js whatever you have and download the latest and stable version according to your device architecture from here -> https://nodejs.dev/en/download/ then restart your pc and it will work fine.

**Comment by DoungChan** [comment]
PS C:\Users\Doung Chan\Desktop\promokh> yarn dev
yarn run v1.22.10
warning ..\..\package.json: No license field
$ next dev
- ready started server on 0.0.0.0:3000, url: http://localhost:3000
warning ..\..\package.json: No license field
- info Using wasm build of next-swc
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
PS C:\Users\Doung Chan\Desktop\promokh> 

I try to install lastest node version but still got an error

**Reply by CutImger** [reply]
+1.

**Comment by ht-l1** [comment]
hey I tried what DanteTerry suggested but still does not work. any other ways (npm) to work around it?
